src/components/contact/assets/contact.scss

```diff
@@ -0,0 +1,49 @@
+@import '~theme/base/variables';
+
+section.App__Contact {
+  display: flex;
+  margin: 40px;
+  
+  
+  .App__LeftSection {
```

Comment:
No need to use BEM, it really doesn't hurt given the production bundle will contain a class name like `.abc123` instead of `App__LeftSection`.

Also, please move `App__LeftSection ` outside of `section.App__Contact`. This will make our css bundle smaller

Reply:
Done as per previous comments.